src/commands/Info.ts

```diff
@@ -19,7 +19,9 @@ export default class Info extends Command
 
 	public action(message: Message, args: Array<string | number>, mentions: User[], original: string): any
 	{
-		message.delete();
+		const prefix: string = !dm ? message.guild.storage.getSetting('prefix') : '';
+		    
+		message.delete();	
```

Comment:
Go ahead and remove the `message.delete()` for me. That wasn't meant to be in there, anyway since it was part of my selfbot code and I don't want the modbot making channels jump around with deletions. 🆗